content/jobstos.md

```diff
@@ -0,0 +1,69 @@
+---
+title: PyBaMM Job Board Terms and Conditions
+shortcutDepth: 1
+summary: "Terms and conditions for posting jobs on the PyBaMM Jobs Board"
+---
+
+## Introduction
+
+These Terms and Conditions govern the job posting services offered by PyBaMM, a project under [NumFOCUS, Inc.](https://numfocus.org). Companies agree to pay a fee for job postings related to battery modeling positions on PyBaMM’s job board. All income generated from this service is considered earned income and is mission-aligned.
+
+---
+
+## Payment for Job Postings
+
+A fee of **$250** will be charged for each job posting. Academic positions (e.g., postdoctoral research assistants) will be charged a discounted rate of **$125**, and PhD positions a further discounted rate of **$62.5**.
+
+Payments will be processed via the [PyBaMM website](https://www.pybamm.org). Upon payment, the job will be posted for a duration of up to **90 days** (or until the closing date, whichever is earlier).
+
+Optionally, companies may make an additional **suggested donation** to support PyBaMM’s ongoing work.
+
+---
+
+## Job Posting Guidelines
+
+- Job advertisements must be relevant to **battery modeling** or related fields.
+- PyBaMM reserves the right to review, approve, or reject any job post to ensure it aligns with its mission.
+
+Approved job postings will remain on the job board for up to **90 days**.
+
+---
+
+## Refund Policy
+
+Payments for job postings are **non-refundable** unless PyBaMM is unable to post the job due to internal issues. In such cases, the Company may request a **full refund**.
+
+---
+
+## Data and Intellectual Property
+
+Any data collected from job applications will be managed per PyBaMM and [NumFOCUS’s privacy policies](https://numfocus.org/privacy-policy). PyBaMM retains ownership of its website content, and companies agree not to post any proprietary information without the appropriate rights.
+
```

Comment:
```suggestion
Any data collected from job applications will be managed by PyBaMM and per [NumFOCUS’s privacy policies](https://numfocus.org/privacy-policy). PyBaMM retains ownership of its website content, and companies agree not to post any proprietary information without the appropriate rights.
```

A minor suggestion that might be unsolicited: the phrase "per PyBaMM" makes it sound like we have a privacy policy of our own in our main repository or in our governance files, which we currently don't